squad/frontend/templates/squad/compare_builds.jinja2

```diff
@@ -36,10 +36,20 @@
               <select id="target-select" name="target" placeholder="{{ _('Enter target version') }}" class="form-control"></select>
             </div>
           </div>
+		  <div class="form-group">
+			<label>{{ _('Comparison type') }}</label>
+			<div>
+			  <input id="test-comparison" name="comparison_type" type="radio" value="test" checked="checked"/>
+			  <label for="test-comparison">{{ _('compare by tests') }}</label>
+			  <br />
+			  <input id="metric-comparison" name="comparison_type" type="radio" value="metric" />
+			  <label for="metric-comparison">{{ _('compare by metrics') }}</label>
+			</div>
+		  </div>
```

Comment:
the indentation here is wrong

Reply:
hm, I my vim got confused with tabs. Fixed that, thanks for spotting that.